web/themes/new_weather_theme/templates/partials/today-summary-forecast.html.twig

```diff
@@ -1,5 +1,5 @@
 {% set today = weather.daily[0] %}
-<h3 class="wx-visual-h2 text-normal text-primary-darker padding-x-2 tablet:padding-x-0 margin-top-3 margin-bottom-2">
+<h3 class="wx-visual-h2 text-normal text-primary-darker padding-x-2 tablet:padding-x-0 margin-top-0 margin-bottom-2">
```

Comment:
This is good for desktop, but realized this needs the 3 units of margin on mobile, otherwise it gets too close to the current conditions section so for this one `margin-top-3 desktop:margin-top-0`